client/src/components/pages/database/license-type.tsx

```diff
@@ -1,11 +1,79 @@
 import "./license-type.css";
+import { FormattedMessage } from "react-intl";
 
 const LicenseType = () => {
   return (
-    <section className="license-type">
-      License Type Section
+    <section className="license-type p-[32px] bg-[#FDFDFD]">
+      <h2 className="font-bold text-[36px] m-0">
+        <FormattedMessage
+          id="license-type.header2"
```

Comment:
You have `header2` here, but `h2` in the json.
between the two, I would keep header2, as it will be more readable for translators.